Cargo.toml

```
@@ -11,12 +11,17 @@ keywords = ["wolfram", "wolfram-language", "mathematica", "wolfram-engine", "exp
 categories = ["encoding"]
 
 [features]
-default = []
+default = ["serde"]
 
 # Whether to publically export nom functions for parsing symbols. This feature should not
 # be considered stable -- it is included only so that wl-parse can build higher-level
 # expression parsing on top of it.
 unstable_parse = []
 
 [dependencies]
+serde = { version = "1.0.*", optional = true }
```

I recommend this be the newest serde version at the time the serde features are added. There might be features in serde 1.0.152 that you're implicitly depending on. 

```suggestion
serde = { version = "1.0.152", optional = true }
```